src/api/categories/__tests__/categories.test.js

```diff
@@ -0,0 +1,240 @@
+import { categories as resource } from '../../resources';
+
+let authorization;
+let id;
+let textId;
+
+const payload = {
+  usualNameFr: 'Catégori',
```

Comment:
I would enjoy to have `Catégorie` instead of `Catégori`.

Reply:
Haha it's a mistake! so it's updated with the patch route :D

Reply:
Oh I see. Too smart for me :smile: